hpi-schul-cloud/schulcloud-server
# BC-6820 - Tldraw file deletion cronjob

# Description

QA note:
Change tested on the env:
https://bc-6820-tldraw-assets-cronjob.nbc.dbildungscloud.dev/dashboard
No issues found. 
More info see tickets: 
https://ticketsystem.dbildungscloud.de/browse/BC-6821
https://ticketsystem.dbildungscloud.de/browse/BC-6820



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-6820


## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
